### PR TITLE
sqlmap: modernize and remove outdated dependencies

### PR DESCRIPTION
**Pull request by youtux**
I've rewritten the formula, since all the dependencies are not needed anymore (they are included in the repository). Furthermore sqlmap developers claim that it's compatible with python 2.6 and 2.7, so there's no need of the python dependency I think.

Should I include a caveat that recommend the user to install sqlmap with the `--HEAD` switch? Because this is the preferred way according to the developers, and sqlmap 0.9 is pretty outdated (April 2011) and lacks a lot of features.

### CODE REVIEW

**Comment by tdsmith** [comment]
Cool! Any chance for a test that does something with e.g. a sqlite database?

**Comment by youtux** [comment]
Done. Suggestions for better code are much appreciated :). Unfortunately it doesn't work with the `--HEAD` version because brew can't correctly locate the binary.

File: Library/Formula/sqlmap.rb [hunk 1]
```diff
@@ -7,76 +5,32 @@ class Sqlmap < Formula
...
-    res = %w{ibm-db impacket ntlm pysqlite}
-    res << "mysql-python" if build.with? "mysql"
-    res << "psycopg2" if build.with? "postgresql"
-    res << "pyodbc" if build.with? "unixodbc"
+    libexec.install Dir["*"]
 
-    res.each do |r|
-      resource(r).stage { system "python", *install_args }
-    end
+    bin.install_symlink libexec/"sqlmap.py"
```
Comment (mikemcquaid) [inline Library/Formula/sqlmap.rb]: Should we maybe just name the original symlink `sqlmap`?

File: Library/Formula/sqlmap.rb [hunk 2]
```diff
@@ -7,76 +5,32 @@ class Sqlmap < Formula
...
+
+    query_select = "select name, age from students order by age asc;"
+
+    # Create the test database
+    `sqlite3 < #{query_path} school.sqlite`
+
+    output = `#{bin}/sqlmap --batch -d "sqlite://school.sqlite" --sql-query "#{query_select}"`
+    assert_match /Bob,\s14/, output
+    assert_match /Sue,\s12/, output
+    assert_match /Tim,\s13/, output
```
Comment (mikemcquaid) [inline Library/Formula/sqlmap.rb]: :heart_eyes:

File: Library/Formula/sqlmap.rb (hunk 1 above)
Comment (youtux) [inline Library/Formula/sqlmap.rb]: Yes we could, the fact is that the head version include also a script `sqlmapapi.py`, and having just `sqlmap` and `sqlmapapi.py` in the bin path would be inconsistent, IMO.

Comment (mikemcquaid) [inline Library/Formula/sqlmap.rb]: Yeh, could do the same thing for the HEAD script too.

Comment (youtux) [inline Library/Formula/sqlmap.rb]: (y)

**Comment by youtux** [comment]
@mikemcquaid @tdsmith Please revise the caveat message, and if you think it's inappropriate I'll remove it.

**Comment by mikemcquaid** [comment]
@youtux Yeh, just remove it I reckon, thanks.

**Pull request closed without merging**